README.md

```diff
@@ -17,6 +17,18 @@ Why is there a first-boot.sh script and a kubevirt-installer.service file? When
 Once the build finishes, note the AMI id. We will use it in the verification step.
 In the case of Gcp, you can set the image name in the environment file
 
+## Making Gcp image publicly available
+
+The following command is used to make the constructed image publicly available
+
+```
+PROJECT="cnvlab-209908"
+BUCKET="kubevirt-button"
+IMAGE="kubevirt-button"
```

Comment:
We will need to follow the same convention Joe established for AWS images. I believe it is kubevirt-centos-pr-X, etc.. I'm sure you guys will do the right thing when GCP is integrated into the pipeline.